sympy/integrals/tests/test_laplace.py

```diff
@@ -142,24 +141,35 @@ def test_laplace_transform():
     assert (LT(sinh(2*sqrt(a*t)), t, s) ==
             (sqrt(pi)*sqrt(a)*exp(a/s)/s**(S(3)/2), 0, True))
     assert (LT(sqrt(t)*sinh(2*sqrt(a*t)), t, s, simplify=True) ==
-            ((-sqrt(a)*s**(S(5)/2) + sqrt(pi)*s**2*(2*a + s)*exp(a/s) *
-              erf(sqrt(a)*sqrt(1/s))/2)/s**(S(9)/2), 0, True))
+            ((sqrt(a)*sqrt(s) + sqrt(pi)*(2*a + s)*exp(a/s)*
+              erf(sqrt(a)/sqrt(s))/2)/s**(S(5)/2), 0, True))
     assert (LT(sinh(2*sqrt(a*t))/sqrt(t), t, s) ==
-            (sqrt(pi)*exp(a/s)*erf(sqrt(a)*sqrt(1/s))/sqrt(s), 0, True))
+            (sqrt(pi)*exp(a/s)*erf(sqrt(a)/sqrt(s))/sqrt(s), 0, True))
     assert (LT(sinh(sqrt(a*t))**2/sqrt(t), t, s) ==
-            (sqrt(pi)*(exp(a/s) - 1)/(2*sqrt(s)), 0, True))
+            (sqrt(pi)*(2*exp(a/s) - 2)/(4*sqrt(s)), 0, True))
     assert (LT(t**(S(3)/7)*cosh(a*t), t, s) ==
             (((a + s)**(-S(10)/7) + (-a+s)**(-S(10)/7))*gamma(S(10)/7)/2,
              a, True))
     assert (LT(cosh(2*sqrt(a*t)), t, s) ==
-            (sqrt(pi)*sqrt(a)*exp(a/s)*erf(sqrt(a)*sqrt(1/s))/s**(S(3)/2) +
-             1/s, 0, True))
+            (2*((sqrt(a)*(sqrt(pi)*(2 - erfc(sqrt(a)/sqrt(s)))
+            + sqrt(s)*exp(-a/s)/sqrt(a))/(4*s**(S(3)/2))
+            + sqrt(a)*(-sqrt(pi)*erfc(sqrt(a)/sqrt(s))
+            + sqrt(s)*exp(-a/s)/sqrt(a))/(4*s**(S(3)/2)))*exp(a/s)), 0, True))
     assert (LT(sqrt(t)*cosh(2*sqrt(a*t)), t, s) ==
-            (sqrt(pi)*(a + s/2)*exp(a/s)/s**(S(5)/2), 0, True))
+            ((((sqrt(pi)*(2 - erfc(sqrt(a)/sqrt(s)))
+            + sqrt(s)*exp(-a/s)/sqrt(a))*(2*a/s + 1)/2
+            - (-sqrt(pi)*erfc(sqrt(a)/sqrt(s))
+            + sqrt(s)*exp(-a/s)/sqrt(a))*(2*a/s + 1)/2)*exp(a/s)
+            - sqrt(pi))/(4*s**(S(3)/2)) + (((sqrt(pi)*(2 - erfc(sqrt(a)/sqrt(s)))
+            + sqrt(s)*exp(-a/s)/sqrt(a))*(2*a/s + 1)/2
+            - (-sqrt(pi)*erfc(sqrt(a)/sqrt(s))
+            + sqrt(s)*exp(-a/s)/sqrt(a))*(2*a/s + 1)/2)*exp(a/s)
+            + sqrt(pi))/(4*s**(S(3)/2)), 0, True))
```

Comment:
In some cases the changed output here is simpler but this case shows an example that is more complex. We need to verify correctness first and then think about improving simplicity afterwards.

Reply:
In this example the output can simplify almost down to the previous result:
```python
In [10]: expr = (((sqrt(pi)*(2 - erfc(sqrt(a)/sqrt(s)))
    ...:             + sqrt(s)*exp(-a/s)/sqrt(a))*(2*a/s + 1)/2
    ...:             - (-sqrt(pi)*erfc(sqrt(a)/sqrt(s))
    ...:             + sqrt(s)*exp(-a/s)/sqrt(a))*(2*a/s + 1)/2)*exp(a/s)
    ...:             - sqrt(pi))/(4*s**(S(3)/2)) + (((sqrt(pi)*(2 - erfc(sqrt(a)/sqrt(s)))
    ...:             + sqrt(s)*exp(-a/s)/sqrt(a))*(2*a/s + 1)/2
    ...:             - (-sqrt(pi)*erfc(sqrt(a)/sqrt(s))
    ...:             + sqrt(s)*exp(-a/s)/sqrt(a))*(2*a/s + 1)/2)*exp(a/s)
    ...:             + sqrt(pi))/(4*s**(S(3)/2))

In [11]: radsimp(expr).factor()
Out[11]: 
                      a
                ____  ─
               ╱  5   s
√π⋅(2⋅a + s)⋅╲╱  s  ⋅ℯ 
───────────────────────
            5          
         2⋅s  
```
The previous result was
```python
In [12]: sqrt(pi)*(a + s/2)*exp(a/s)/s**(S(5)/2)
Out[12]: 
            a
            ─
   ⎛    s⎞  s
√π⋅⎜a + ─⎟⋅ℯ 
   ⎝    2⎠   
─────────────
     5/2     
    s   
```
These are equivalent if `s^(5/2)` is equivalent to `s^5/sqrt(s^5)` but that doesn't hold. The SymPy interpretation of `s^(5/2)` is `(s^(1/5))^2` rather than `(s^5)^(1/2)` so these are not equivalent.